bin/truffleruby

```diff
@@ -99,6 +122,9 @@ if [ $on_graalvm = false ]; then
 
 fi
 
+# no " to split $JAVA_OPTS into array elements
+java_args+=($JAVA_OPTS)
```

Comment:
Is -J-cmd used in JAVA_OPTS?
Currently it's not supported but I am not sure it's real issue in practice.
I never did `JAVA_OPTS=-cmd ...` as `-cmd` is not a flag `bin/java` recognizes.
I could see a use-case for -cp, but that would require special handling and I think out of scope of this PR.